src/main/kotlin/net/atos/zac/app/zaak/model/RESTResultaattype.kt

```diff
@@ -29,3 +35,25 @@ data class RESTResultaattype(
 
     var selectielijst: String? = null,
 )
+
+fun ResultaatType.toRestResultaatType() = RESTResultaattype(
+    id = UriUtil.uuidFromURI(this.url),
+    naam = this.omschrijving,
+    toelichting = this.toelichting,
+    archiefNominatie = this.archiefnominatie.name,
+    archiefTermijn = this.archiefactietermijn?.let {
+        PeriodUtil.format(Period.parse(it))
+    },
+    besluitVerplicht = this.brondatumArchiefprocedure?.afleidingswijze?.let {
+        Afleidingswijze.VERVALDATUM_BESLUIT.toValue() == it.name.uppercase(Locale.getDefault()) ||
```

Comment:
Isn't default locale used by default. Seem too explicit to me

Reply:
No, it is a mandatory argument. You have to specify the locale explicitly. see: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/